server/bin/pbench-server-put-shim.py

```diff
@@ -0,0 +1,71 @@
+#!/usr/bin/env python3
+# -*- mode: python -*-
+
+# This script is used to send the tarballs that a version 002 client
+# submits for further processing. It forwards the tarballs and their MD5
+# sums to the remote server using the server's HTTP PUT method
+
+import os
+import sys
+import tempfile
+
+from pbench import PbenchConfig, BadConfig, get_pbench_logger
+from pbench.api.process_tb import ProcessTb
+from pbench.report import Report
+
+
+_NAME_ = "pbench-server-put-shim-001"
```

Comment:
We're not including the `-001` suffix on the file name, and hopefully there'll never be a need for a `002`, so let's just drop that from the name.

Reply:
oh yes. I missed that